api/ReadableStreamBYOBReader.json

```
@@ -266,13 +266,27 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/ReadableStreamBYOBReader/releaseLock",
           "spec_url": "https://streams.spec.whatwg.org/#ref-for-byob-reader-release-lock②",
           "support": {
-            "chrome": {
-              "version_added": "89"
-            },
+            "chrome": [
+              {
+                "version_added": "105"
+              },
+              {
+                "version_added": "89",
+                "partial_implementation": true,
+                "notes": "Before version 105, `releaseLock()` would throw if there are pending read requests (rather than pending read requests being rejected)."
+              }
```

```suggestion
              {
                "version_added": "89",
                "version_removed": "105",
                "partial_implementation": true,
                "notes": "The `releaseLock()` method would throw if there are pending read requests (rather than pending read requests being rejected)."
              }
```

use the  *version_removed* field instead of mentioning it in the notes *field*

same as above